CHANGELOG.md

```diff
@@ -140,6 +140,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Fixed support for `ModelCheckpoint` monitors with dots ([#12783](https://github.com/PyTorchLightning/pytorch-lightning/pull/12783))
 
+- Fixed printing of ragged dictionaries in `EvaluationLoop._print_results` ([#12857](https://github.com/PyTorchLightning/pytorch-lightning/pull/12857))
```

Comment:
In the changelog, we should mention the user facing api whenever possible, i.e., Trainer.validate/test instead of the internal methods.

Reply:
Changed